test/e2e/framework/skipper/skipper.go

```diff
@@ -213,6 +213,20 @@ func SkipUnlessNodeOSDistroIs(supportedNodeOsDistros ...string) {
 	}
 }
 
+// SkipUnlessMasterOSArchIs skips if the master OS distro is not included in the supportedMasterOsArchs.
+func SkipUnlessMasterOSArchIs(supportedMasterOsArchs ...string) {
```

Comment:
This SkipUnlessMasterOSArchIs() is not used in this PR.
How about just implementing SkipUnlessNodeOSArchIs() only without SkipUnlessMasterOSArchIs()?
(I know this can be implemented for consistency like the other functions)

Now we are trying to avoid using code related to master node thing like https://github.com/kubernetes/kubernetes/issues/80177
So if this is not mandatory, it is nice to skip implementing.

Reply:
I have no objection. I actually think at some point some of these things could be read from node labels.

Reply:
Done.